### PR TITLE
38 explore llama gemini

### LINKED ISSUE #38: Explore Llama & Gemini
Let's use this library
https://github.com/andrewyng/aisuite

### PR DESCRIPTION
**Pull request by samerrak**
used ai-suite to transform llm calls

### CODE REVIEW

**Comment by samerrak** [comment]
updated

**Review by evanboyd31 — CHANGES_REQUESTED** [review]
Had to fix some imports but it appears to be working well for calling the different LLMs. It seems that the the other LLMs cannot produce some tables (for instance the events table) correctly. This is because some of the actions had their old prompts removed. Could you add those back and lmk?

**Review by evanboyd31 — APPROVED** [review]
:100: 

Didn't test entire runs for the event-driven SMF bc I didn't want to hit the quota but the initial outputs look fantastic. This is good merge once all the binary files are removed. Awesome work!

File: backend/resources/util.py [hunk 1]
```diff
@@ -1,11 +1,55 @@
...
+                return "google:gemini-1.5-pro-001"
+            else:
+                print("Invalid choice. Please enter 1, 2, 3 or 4.")
+        except ValueError:
+            print("Invalid input. Please enter a number (1, 2, 3, or 4).")
+
+model = choose_model()
+
+def call_llm(prompt, max_tokens=1200, temperature=0.7):
+=======
```
Comment (evanboyd31) [inline backend/resources/util.py]: Looks like some merge conflicts got messed up here. Please fix this, test the SMFs to make sure the `call_llm` function works, and make sure the environmental impact calculations display at the end

**Review by evanboyd31 — CHANGES_REQUESTED** [review]

**Review by evanboyd31 — APPROVED** [review]

**Pull request merged**